types/linter.d.ts

```diff
@@ -30,21 +30,21 @@ declare namespace linter {
     LintResultMap,
   };
 }
-type Options = import('./options').Options;
-type Compilation = import('webpack').Compilation;
-type Linter = (files: string | string[]) => void;
-type Reporter = () => Promise<Report>;
 type ESLint = import('eslint').ESLint;
 type Formatter = import('eslint').ESLint.Formatter;
 type LintResult = import('eslint').ESLint.LintResult;
 type Compiler = import('webpack').Compiler;
+type Compilation = import('webpack').Compilation;
+type Options = import('./options').Options;
```

Comment:
why did you move the order here?

Reply:
I think it done typescript because of update, they are autogenerated